### DISCUSSION TITLE
posts/Chirpy-Blog-Customization/

### DISCUSSION THREAD

**Discussion by giscus** [huanyushi/huanyushi.github.io#1] [Announcements]
# posts/Chirpy-Blog-Customization/

1. 简介 在去年利用 jekyll 在 github 部署静态博客网站，效果甚合我意。

https://huanyushi.github.io/posts/Chirpy-Blog-Customization/

**Comment by CHENHUI-X** [comment]
好好好 大佬又发新的了 , 我又可以抄了 ~~~

**Reply by Hisqkq** [reply]
Awesome work thank you !

**Comment by cseek** [comment]
大佬牛逼啊，我搬一点回家

**Comment by tortrixx** [comment]
感谢大佬的教程，附一张ai总结图：
<img width="2848" height="1504" alt="Gemini_Generated_Image_pwycodpwycodpwyc" src="https://github.com/user-attachments/assets/b9482c23-f6e3-4021-8f80-5056f791ee7c" />

**Comment by Croft233** [comment]
tql,学习到了
